support-models/src/test/scala/com/gu/support/promotions/SerialisationSpec.scala

```diff
@@ -111,9 +115,11 @@ class SerialisationSpec extends AsyncFlatSpec with SerialisationTestHelpers with
       appliesTo,
       "",
       Map[Channel, Set[PromoCode]]("" -> Set()),
-      DateTime.now(),
+      DateTime.now(ISOChronology.getInstanceUTC),
       None, None, None
     )
+    implicit val encodeDateTime: Encoder[DateTime] = Encoder.encodeString.contramap(ISODateTimeFormat.dateTime().print)
```

Comment:
before this change the test was incorrectly encoding as a Long because it had picked up the encoder we use for sending dates to the client side.  Other tests were covering the correct format, and I'm not even sure if this test is useful, but I've fixed it up anyway.